source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+Otherwise if you don't have one of these two very specific formats, you'll need to make the manifest file to give ``import`` instructions on what and how to import your files.
+
+Demultiplexing sequences
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have reads from multiple samples in the same file, you'll need to demultiplex your sequences.
+
+If your barcodes have already been removed from the reads and are in a separate file, you can use :doc:`q2-demux <../plugins/available/demux/emp-paired/>` to demultiplex these.
+
+If your barcodes are still in your sequences, you can use functions from the :doc:`cutadapt
+plugin <../plugins/available/cutadapt/index>`.
+The ``cutadapt demux-single`` method looks for barcode sequences at the beginning of your reads (5' end) with a certain error tolerance, removes them, and returns sequence data separated by each sample.
+The QIIME 2 forum has a `tutorial on various functions available in cutadapt <https://forum.qiime2.org/t/demultiplexing-and-trimming-adapters-from-reads-with-q2-cutadapt/2313>`__, including demultiplexing.
+You can learn more about how ``cutadapt`` works under the hood by reading their `documentation <https://cutadapt.readthedocs.io/en/stable/index.html>`__.
+
+Note: Currently ``q2-demux`` and ``q2-cutadapt`` do not support demultiplexing dual-barcoded paired-end sequences, but only can demultiplex with barcodes in the forward reads.
+So for the time being for this type of demultiplexing needs to be done outside of QIIME 2 using other tools, for example
+`bcl2fastq <https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html>`__.
+
+.. _`merge reads`:
+
+Merging reads
+~~~~~~~~~~~~~~
+
+Whether or not you need to merge reads depends on how you plan to cluster or denoise your sequences into amplicon sequence variants (ASVs) or operational taxonomic units (OTUs).
+
+If you do need to merge your reads, you can use the QIIME 2 :doc:`VSEARCH plugin <../plugins/available/vsearch/index>`  with the :doc:`join-pairs <../plugins/available/vsearch/join-pairs/>` method.
+
+.. _`Remove non-biological sequences`:
+
+Removing non-biological sequences
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your data contains any non-biological sequences (e.g. primers, sequencing adapters, PCR spacers, etc), you should remove these.
+
+The :doc:`q2-cutadapt <../plugins/available/cutadapt/index>` plugin has comprehensive methods for removing non-biological sequences from :doc:`paired-end <../plugins/available/cutadapt/trim-paired/>` or :doc:`single-end <../plugins/available/cutadapt/trim-single/>` data.
+
+If you're going to use DADA2 to denoise your sequences, you can remove biological sequences at the same time as you call the denoising function.
+All of DADA2's ``denoise`` fuctions have some sort of ``--p-trim`` parameter you can specify to remove base pairs from the end(s) of your reads.
+(Deblur does not have this functionality yet.)
```

Comment:
Add this?
"Note that dada2's trim parameter only trims the 5' end. Although the `--p-truncate` parameter can be used similarly to trim the 3' ends of sequences, read length heterogeneity can make it very difficult to predict the appropriate truncation site for removing non-biological sequence data. In this situation, it may be preferable to use q2-cutadapt to trim non-biological sequence data prior to using dada2."

Reply:
Ok, I added the clarification that `--p-trim-left` only trims the 5' end.

But I'm confused about the `--p-truncuate` statement here - why should people use cutadapt to truncate reads rather than passing in an appropriate parameter to `--p-truncate`? Seems to me like the workflow that makes the most sense is to keep things simple and do things all within dada2...?

Reply:
Yep DADA2 is usually enough to handle both cases.

cutadapt is mostly for the case where your reads will run through the reverse-complemented opposing-primer (common with ITS). It isn't something you'd worry about with 515-806, but other primer sets can have the problem.

Reply:
ok, i won't add any mention of the `--p-truncate` parameter here because we discuss it below in the dada2 part :)